eng/Generate.ps1

```diff
@@ -1,6 +1,8 @@
-param($name, [switch]$noDebug)
+param($name, [switch]$noDebug, [switch]$NoReset)
 $ErrorActionPreference = 'Stop'
 
+$AutorestCoreVersion = "3.0.6162"
```

Comment:
This should be part of the readme.md of the extension (file in root). Then, the extension itself depends on a specific version of the core, which is what we want when someone uses our extension.

Reply:
Fixed